go/src/koding/config/genconfig.go

```diff
@@ -0,0 +1,233 @@
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+)
+
+var (
+	input  = flag.String("i", "-", "")
+	output = flag.String("o", "-", "")
+)
+
+var t = template.Must(template.New("").Parse(`// This file is auto-generated. DO NOT EDIT!
```

Comment:
i still cant follow the idea, sorry :/ 

What is the problem you see creating a json/toml from the coffee and using multiconfig to read that and using `required` tag?

It is simple, it is correct, it is maintainable - we are using it in all of our workers (> 25 workers)